### ISSUE TITLE
Are there related papers for this project?

### ISSUE THREAD

**Issue by Thinktan** [sawankumar94/Key-Frame-Extraction-from-Videos#1]

**Comment by Thinktan** [comment]
@sawankumar94

**Comment by sawankumar94** [comment]
Yes, There are related papers for this project. I didn't save links to those papers but I should have PDF files. I am sorry for delay in the response.

**Comment by Thinktan** [comment]
Can you share this pdf doc? thanks you very much @sawankumar94

**Comment by sawankumar94** [comment]
[Research Paper 2.pdf](https://github.com/sawankumar94/Key-Frames-Extraction-from-Video/files/5841823/Research.Paper.2.pdf)
[Research Paper 1.pdf](https://github.com/sawankumar94/Key-Frames-Extraction-from-Video/files/5841826/Research.Paper.1.pdf)
[Deep_Keyframe_Detection_in_Human_Action_Videos.pdf](https://github.com/sawankumar94/Key-Frames-Extraction-from-Video/files/5841827/Deep_Keyframe_Detection_in_Human_Action_Videos.pdf)

Hi, 
Please find the PDFs of research papers in the attachment. Happy reading.
